src/main/java/querqy/elasticsearch/rewriterstore/DeleteRewriterResponse.java

```diff
@@ -34,7 +34,6 @@ public void writeTo(final StreamOutput out) throws IOException {
         clearRewriterCacheResponse.writeTo(out);
     }
 
-    @Override
```

Comment:
This has a smell. If we can remove `Override`, this method is no longer being used. Can we remove it? Also, maybe the super class is now providing a new method that replaces `status()` and that we should override now?

Reply:
I thought so too. But it looks to me like the interface method was simply removed.